extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -345,6 +347,45 @@ export class AmpA4A extends AMP.BaseElement {
 
     /** @private {string} */
     this.safeframeVersion_ = DEFAULT_SAFEFRAME_VERSION;
+
+    /** @private @const {RefreshManager}  */
+    this.refreshManager_ = getRefreshManagerFor(this.win);
+
+    this.refreshManager_.registerElement(this.element, refresher => {
+      console.log('Eureka!');
+      this.isRefreshing_ = true;
+      this.unlayoutCallback();
```

Comment:
I factored out the part of unlayoutCallback that destroys state related to the frame into a separate function. When refreshing, the frame is not removed when unlayoutCallback is invoked.